requirements.txt

```diff
@@ -1,4 +1,4 @@
-boto3>=1.10.0
-cryptography>=2.5.0
-attrs>=17.4.0
-wrapt>=1.10.11
+boto3==1.20.24
+cryptography==36.0.1
+attrs==21.2.0
+wrapt==1.13.3
```

Comment:
I think that this is a bad idea. Why? Our customers now cannot use different versions of these dependencies in their environments. I recommend reverting the changes in this file.